packages/front-end/enterprise/components/Dashboards/DashboardEditor/DashboardBlock/DescriptionBlock.tsx

```diff
@@ -0,0 +1,30 @@
+import React from "react";
+import { DescriptionBlockInterface } from "back-end/src/enterprise/validators/dashboard-block";
+import { ScrollArea } from "@radix-ui/themes";
+import Markdown from "react-markdown";
+import { Box } from "spectacle";
+import { BlockProps } from ".";
+
+export default function DescriptionBlock({
```

Comment:
Rename to ExperimentDescription (and underlying type field)? Other things that will soon appear on dashboards will also have descriptions